ozy/installers/conda.py

```diff
@@ -11,7 +11,6 @@
 
 
 def do_conda_install(conda_bin, channels, to_dir, to_install):
-    os.makedirs(to_dir)
```

Comment:
Womp womp. This needs to make the parent directory since we use it below to make a temporary directory next to the target environment dir.

`Path(to_dir).parent.mkdir(exists_ok=True, parents=True)`

Reply:
yeah - figured that might also work - will make that change